docs/source/tutorials.rst

```diff
@@ -0,0 +1,7 @@
+Tutorials
```

Comment:
As mentioned above, maybe rename it to "Additional Tutorials"

Reply:
Done